operatorcourier/push.py

```diff
@@ -45,14 +51,16 @@ def _create_base64_bundle(self, bundle_dir, repository):
             result64 = base64.b64encode(result).decode("utf-8")
             return result64
 
-    def _push_to_registry(self, namespace, repository, release, bundle, auth_token):
-        push_uri = 'https://quay.io/cnr/api/v1/packages/%s/%s' % (namespace, repository)
+    def _push_to_registry(self, namespace, repository, release, bundle, auth_token,
+                          quay_host, verify_host):
+        push_uri = 'https://%s/cnr/api/v1/packages/%s/%s' % (quay_host, namespace,
```

Comment:
If we are going to do this, rather than just support additional `quay.io/cnr` endpoints, why not just make the entire root path of the host to the API configurable? i.e.:

```
push_uri = 'https://%s/api/v1/packages/%s/%s' % (host, namespace,repository)
```

`https://quay.io/cnr` is the full arbitrary root path, the `cnr` is not part of the app-registry spec. Here's a full client implementation for reference: https://github.com/operator-framework/go-appr